src/audio/sink_api_helper.c

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2023 Intel Corporation. All rights reserved.
+//
+
+#include <sof/audio/sink_api.h>
+#include <sof/audio/sink_api_internal.h>
+
+inline void sink_init(struct sink *sink, const struct sink_ops *ops,
+		      struct sof_audio_stream_params *audio_stream_params)
```

Comment:
if possible `const`

Reply:
no, const is not an option as there are SET functions in API